src/split-pane/index.ts

```diff
@@ -55,6 +56,23 @@ export class SplitPaneBase<P extends SplitPaneProperties = SplitPaneProperties>
 	private _lastSize?: number;
 	private _position = 0;
 	private _collapsed = false;
+	private _resizeResultOverridden = false;
+	private _width = 0;
+
+	@diffProperty('collapseWidth')
+	@diffProperty('direction')
+	private collapseWidthDiff(oldProperties: SplitPaneProperties, { collapseWidth, direction, onCollapse }: SplitPaneProperties) {
```

Comment:
It looks like you're effectively removing the default `collapseWidth` value of 600 that was set in the old `diffProperty`s. Is that a breaking change?

Reply:
again, good eye, thanks